### DISCUSSION TITLE
Share a link to search results on my site?

### DISCUSSION THREAD

**Discussion by brabster** [squidfunk/mkdocs-material#7636] [Q&A] [answered]
Can I share a link to search results? Something like https://tempered.works?search=cdc

URL doesn't change when I search for something, haven't found any settings or mention in docs so far but might be missing something obvious...

**Comment by squidfunk** [comment, marked as answer]
This is already available ☺️
https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/?q=search+sharing

**Reply by brabster** [reply]
Awesome thank you very much!
